ql/src/java/org/apache/hadoop/hive/ql/metadata/SessionHiveMetaStoreClient.java

```diff
@@ -2506,13 +2506,13 @@ private Map<Object, Object> getQueryCache() {
   }
 
   @Override
-  protected String getValidWriteIdList(String dbName, String tblName) {
+  protected String getValidWriteIdList(String catName, String dbName, String tblName) {
```

Comment:
an alternate approach would be to replace arguments like this with a `TableName`...so that we move forward with that refactor and use that object more extensively; 

I think bugs like this are simply just sideeffects of passing a set of strings together - and sometimes miss one...